### PR TITLE
added UserNotFound component to the Roles page and improved the search

### LINKED ISSUE #613: Bug Report - Search users' list by name doesn't work
**Describe the bug**
When logged in, searching for a user by name does not do anything. 

**To Reproduce**

1. Click on the  "Roles" navlink on the navbar.
2. Click on the input field and begin typing to search

**Expected behavior**
A list of users with names that include the letters searched should show up on the screen.

**Actual behavior**
Nothing happens.

**Screenshots**
talawa-search-by-name

**Additional details**
If it is a case where there is only supposed to be one user (not organization-wise), it will make sense to take the whole search feature away.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Pranauv-Kumar1803**
**What kind of change does this PR introduce?**
Fixed the search by name in Roles page so that now, when there are no users present, it will show 'UserNotFound' instead of giving unnecessary results and added userNotFound component along with its translations to the Roles page.

**Issue Number:**
Fixes #613 

**Did you add tests for your changes?**
Yes.

**Snapshots/Videos:**
https://user-images.githubusercontent.com/97829544/227679036-10cf4a5b-6196-40c3-bd89-e1faf65be6e3.mp4

**If relevant, did you update the documentation?**
Not Relevant.

**Summary**
Made the search by name in Roles section better by making it check for errors from graphql ( given when a user is not found ) and also made the postNotFound component as reusable and implementing it in the Roles section. Also added a new key in locales to convert it into other languages.

**Does this PR introduce a breaking change?**
Small change but better user experience.

**Other information**

None.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   "scripts": {
     "serve": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --watchAll --coverage",
+    "test": "react-scripts test --watchAll",
```
Comment (anshgoyalevil) [inline package.json:66]: @Pranauv-Kumar1803 

Why have you modified the ```package.json``` file?

Comment (anshgoyalevil) [reply package.json:66]: It will cause all of the workflows to fail, since ```lcov.info``` will not be generated in this case.

Comment (Pranauv-Kumar1803) [reply package.json:66]: ok thank you

Comment (Pranauv-Kumar1803) [reply package.json]: Done @anshgoyalevil

Comment (anshgoyalevil) [reply package.json]: Thanks @Pranauv-Kumar1803

**Comment by Pranauv-Kumar1803** [comment]
@palisadoes Pls Review!

**Comment by Pranauv-Kumar1803** [comment]
This is the reopened pr of this #799 as it was closed while syncing fork with the main admin repo!

**Review by Rumone — APPROVED** [review]

**Review by Kevoniat — APPROVED** [review]

**Comment by Pranauv-Kumar1803** [comment]
@Rumone @Kevoniat 
can you please tell me why is there a toast component saying "there are no organizations found" in the Roles page?

**Pull request merged**